src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -355,6 +355,9 @@ protected function describeContainerDefinition(Definition $definition, array $op
             $tableRows[] = ['Arguments', implode("\n", $argumentsInformation)];
         }
 
+        $usages = $this->getUsagesForDefinition($definition);
+        $tableRows[] = ['Used by', \count($usages) > 0 ? implode(', ', $usages) : 'no'];
```

Comment:
```suggestion
        $tableRows[] = ['Used by', $usages ? implode(', ', $usages) : 'no'];
```

No need to count the array just to check for emptiness. Same for the MD descriptor.

Reply:
Fixed!